ldap-client.go

```diff
@@ -41,7 +45,10 @@ func (lc *LDAPClient) Connect() error {
 				return err
 			}
 		} else {
-			l, err = ldap.DialTLS("tcp", address, &tls.Config{InsecureSkipVerify: false})
+			l, err = ldap.DialTLS("tcp", address, &tls.Config{
+				InsecureSkipVerify: lc.InsecureSkipVerify,
```

Comment:
Yep, makes sense :)

This commit effectively provides a mechanism for the user to enable ssl. If they want to keep cert verification they can specify `ServerName` and if they don't, they can disable verification. I suppose this project could set the default `ServerName` based on `Host` while keeping cert verification. The current approach honors the golang montra of "do less enable more" :)